server/models/data/availabilitydetails.js

```diff
@@ -0,0 +1,14 @@
+
+class AvailabilityDetails {
+
+    constructor(availabilityData) {
+        this.schoolId = availabilityData.school_id;
+        this.workerId = availabilityData.worker_id,
+        this.startTime = availabilityData.start_time;
+        this.endTime = availabilityData.end_time;
+        this.date = availabilityData.date;
+        this.worker_timeslot_id = availabilityData.worker_timeslot_id;
```

Comment:
Hi Nancy, 
I would recommend also naming this variable in lowerCamelCase to be consistent with the other formats, e.g., `this.workerTimeslotId = availabilityData.worker_timeslot_id`

Reply:
Hi Amy,
Thank you for catching that!

My action: I've updated the workerID to be in camelCase as follows: `this.workerTimeslotId = availabilityData.worker_timeslot_id`

You can now see this change in line 0 of server/models/data/availabilitydetails.js

Please confirm if this takes care of your suggestion!